src/platforms/common/configuration/releases.mdx

```diff
@@ -53,6 +53,17 @@ Releases are global per organization; prefix them with something project-specifi
 
 </Note>
 
+The behavior of a few features depends on whether a project is using semantic or time-based versioning.
+
+- Regression detection
+- `release:latest`
+
+We automatically detect whether a project is using semantic or time-based versioning based on:
+
+- If ≤ 2 releases total: we look at most recent release.
+- If 3-10 releases total: if any of the most recent 3 releases is semver, project is semver.
```

Comment:
I feel like `3-10` is a bit confusing because it is not right inclusive as in with up to 9 releases this rule applies

Reply:
Does this work? If not, please feel free to suggest an edit.

```suggestion
- If 3-9 releases (inclusive): if any of the most recent 3 releases is semver, project is semver.
```

Reply:
This makes sense to me .. looks good